packages/rocketchat-api/server/v1/permissions.js

```diff
@@ -0,0 +1,17 @@
+/**
+	This API returns all permissions that exists
+	on the server, with respective roles.
+
+	Method: GET
+	Route: api/v1/permissions
+ */
+RocketChat.API.v1.addRoute('permissions', { authRequired: true }, {
+	get() {
+		let result;
+		Meteor.runAsUser(this.userId, () =>
```

Comment:
@MarcosSpessatto could you change this to 

```javascript
const result = Meteor.runAsUser(this.userId, () => Meteor.call('permissions/get'));
```

`Meteor.runAsUser` returns the result of the passed function.